src/encoded/static/components/genome_browser.js

```diff
@@ -475,7 +482,12 @@ class GenomeBrowser extends React.Component {
                         <div ref={(div) => { this.chartdisplay = div; }} className="valis-browser" />
                     </div>
                 :
-                    <div className="browser-error valis-browser">There are no visualizable results.
+                    <div>
+                        {(this.state.disableBrowserForIE) ?
+                            <div className="browser-error valis-browser">The genome browser does not support Internet Explorer. Please upgrade your browser to Edge in order to visualize files on ENCODE.</div>
```

Comment:
This is a pure taste thing, but I usually find I can remove “in order” and keep the same meanning and clarity in two fewer words.